src/ucp/core/ucp_request.inl

```diff
@@ -640,13 +640,15 @@ ucp_send_request_next_am_bw_lane(ucp_request_t *req)
     }
 }
 
-static UCS_F_ALWAYS_INLINE uintptr_t ucp_request_get_dest_ep_ptr(ucp_request_t *req)
+static UCS_F_ALWAYS_INLINE ucs_ptr_map_key_t
+ucp_request_send_get_ep_rkey(ucp_request_t *req)
```

Comment:
why `send` in the function name?

Reply:
by analogy with other functions which work only with send requests

Reply:
then `ucp_send_request_get_ep_rkey` 
(like `ucp_send_request_get_am_bw_lane`)